src/pieces/commandManager.ts

```diff
@@ -14,6 +14,13 @@ async function register(bot: Client): Promise<void> {
 		bot.emit('error', error);
 	}
 
+	bot.on('messageCreate', async msg => {
+		if (msg.content.substring(0, 8).includes('s;enroll')) {
+			msg.reply('Please use the dropdowns in #role-select instead!');
+			await msg.delete();
```

Comment:
Might generalize this to be: 
```suggestion
		if (msg.content.substring(0, 8).startsWith('s;')) {
			msg.reply('If you\'re trying to run a Sage command, we\'ve moved over to using slash commands. If you're trying to enroll in a course, please use the dropdowns in #role-select instead!');
			await msg.delete();
```

Reply:
You could also wait like 20 seconds and delete the reply after that too

Reply:
Went with 10 instead because 20 felt like a lot, but done